handbook/engineering/README.md

```diff
@@ -756,14 +756,14 @@ At this state, the [bug review DRI](#rituals) (QA) is responsible for going thro
 #### Weekly bug review
 QA has weekly check-in with product to go over the inbox items. QA is responsible for proposing “not a bug”, closing due to lack of response (with a nice message), or raising other relevant questions. All requires product agreement
 
-Requesters have six weeks to provide follow-up information for each request. We'll ping them again as a reminder at three weeks. After six weeks, we'll close the bug to remove it from our visibility, but requesters are welcome to re-open and provide context.
-
 QA may also propose that a reported bug is not actually a bug. A bug is defined as “behavior that is not according to spec or implied by spec.” If agreed that it is not a bug, then it's assigned to the relevant product manager to determine its priority.
 
 #### Acknowledging bugs
 If the inbox item is a bug, QA should apply the acknowledged state to the bug. QA has one week to reproduce the bug.
 
-Once reproduced, QA should document the reproduction steps and move it to the reproduced state.
+For community-reported bugs, this may require QA to gather more information from the reporter. QA should acknowledge the bug and reach out to reports if more information is needed to reproduce the issue. Reporters have six weeks to provide follow-up information for each report. We'll ping them again as a reminder at three weeks. After six weeks, we'll close the bug to remove it from our visibility, but reporters are welcome to re-open and provide context.
+
+Once reproduced, QA should document the reproduction steps in the description, align with product on what is the desired outcome during the weekly bug review meetings, and move it to the reproduced state.
 
 #### Reproduced
 When reproduced, the bug is assigned to the appropriate EM and added to the product backlog. The EM is responsible for investigating the root cause of the bug and proposing solutions to their product counterpart if it requires discussion. Otherwise, the EM includes it in this release (if there's space) or the next release.
```

Comment:
@xpkoala @zayhanlon @sharon-fdm 

on reproduction steps, I think there is an ambiguity around it. I think there needs to be some communication between QA and the EM to agree on how to reproduce the issue. It's probably driven by QA or the EM and supported by product

Reply:
@zhumo I'm not sure if I understand. Isn't it QA's responsibility to determine how to reproduce the issue and document it? Won't it be different every time based on the issue submitted? Unless I'm not understanding about a standard way / steps to follow for every type of submission

Reply:
My understanding is there is sometimes a gap between what I am assuming is communicated clearly through a ticket description and what information the EM has on the issue and it's severity based on the notes. Does that sound correct @zhumo?

Moving forward I can add reproduction steps to all bug tickets moving from my plate to the EM.

Reply:
We met today and agreed that the flow is: 
- reproduce the issue
- share/describe the reproduction steps to product at the weekly bug review meeting
- Align with product on what is the desired behavior

We expect back and forths between QA and eng will be necessary in a small minority of cases, so that can be done informally. If we find continued problems with acceptance of the reproduction steps, we can revisit the process further.